src/libs/DateUtils.js

```diff
@@ -25,15 +25,20 @@ Onyx.connect({
  * Moment object for the given timestamp
  *
  * @param {String} locale
+ * @param {Object} currentTimezone
  * @param {Number} timestamp
  *
  * @returns  {Moment}
  *
  * @private
  */
-function getLocalMomentFromTimestamp(locale, timestamp) {
+function getLocalMomentFromTimestamp(locale, currentTimezone = {}, timestamp) {
```

Comment:
I think timezone should be added at the end of the parameters, since it's got a default value.